src/settings/MobileAppSettings.js

```diff
@@ -19,6 +20,19 @@ export class MobileAppSettings extends Settings {
     }
   }
 
+  get(key) {
+    switch (key) {
+      case 'Hardware_UUID': return this.getHardwareUUID();
```

Comment:
Hi @raviSussol, I was suggesting that DeviceInfo.getUniqueID() is called only once, on app startup (in MobileAppSettings constructor) and the value is stored in the settings (then retrieved from settings).
Can you please remove the extra `get(key)` and `getHardwareUUID()`. And instead just have one line in the constructor: 
```
  constructor(database) {
    super(database);
    this.load();
    this.set(SETTINGS_KEYS.HARDWARE_UUID, DeviceInfo.getUniqueID())
  }

```

Reply:
`load()` method defined at the bottom was intended for this kind of stuff, though in looking at it I think it's a bit silly, the content of `load` would be fine to have inline in the constructor.

Reply:
@Chris-Petty @andreievg - should I remove `load()` method and replace it by putting inline code inside constructor too?